react/CreditCard.tsx

```diff
@@ -156,8 +157,30 @@ const CreditCard = forwardRef<CreditCardRef, Props>(function CreditCard(
     }
   }, [])
 
+  const updateAddress = (address: string | Address) => {
+    if (!iframeRef.current) {
+      return
+    }
```

Comment:
This will evaluate to true when the iframe isn't loaded, right? If that's true, the iframe will only be unloaded during the page's initial load, correct?

Reply:
sort of, this will be true if we didn't render the iframe yet (imagine calling this on a `useLayoutEffect` where React hasn't commited the changes to the DOM yet), but I don't think we'll ever bump into a scenario where it isn't rendered. I mostly added this because of TypeScript (and to avoid non-null assertions)